htsinfer/get_library_stats.py

```diff
@@ -48,21 +51,31 @@ def evaluate(self) -> ResultsStats:
         stats = ResultsStats()
 
         # process file 1
-        LOGGER.info(f"Obtaining statistics for file: {self.paths[0]}")
-        stats.file_1.read_length.min, stats.file_1.read_length.max = (
-            self.fastq_get_min_max_read_length(fastq=self.paths[0])
+        LOGGER.info("Obtaining statistics for file: %s", self.paths[0])
```

Comment:
Better to be consistent and use an f-string. Also for other log messages below

Reply:
Thanks, fixed it.